src/test/java/homework_2/random_chars_table/RandomCharsTableTest.java

```diff
@@ -0,0 +1,133 @@
+package homework_2.random_chars_table;
+
+import base.UnitBase;
+import org.junit.jupiter.api.RepeatedTest;
+import org.junit.jupiter.api.Test;
+
+import java.util.NoSuchElementException;
+import java.util.Random;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class RandomCharsTableTest extends UnitBase {
+    @Test
+    void InputZeroAsArgument() {
+        setInput("0 0 odd");
+
+        new RandomCharsTable().run();
+        printOut();
+        removeFromOutput("Введите данные:");
+        assertEquals("Passed parameters should match the format [positive integer] [positive integer] [even|odd]", getOutputLines()[0]);
+    }
+
+    @Test
+    void InputEmptyArgument() {
+        setInput("");
+        assertThrows(NoSuchElementException.class, () -> new RandomCharsTable().run());
+    }
+
+    /*
+        for this test use new RandomCharsTable().setDebug(true).run();
```

Comment:
If think, once you'll understand how to check random letters, you'll not need debug mode. Good idea anyway